src/odemis/gui/conf/data.py

```diff
@@ -610,6 +618,13 @@
                 "control_type": odemis.gui.CONTROL_SLIDER,
             },
         },
+        # TODO needed?
```

Comment:
Not needed. This is in case the GUI needs to show directly the settings for the pol-analyzer, which it doesn't.